aten/src/ATen/core/op_registration/op_registration.h

```diff
@@ -662,6 +662,23 @@ class CAFFE2_API CppFunction final {
     );
   }
 
+  // TODO: more user friendly API
+  static CppFunction makeFallthrough() {
```

Comment:
Also: if fallbacks are lower precedence than op-specific kernels, why are fallthroughs needed? Maybe just to avoid empty dispatch table entries?

Reply:
Obsoleted by comment above